frontend/components/swiper/SwiperWrapper.tsx

```diff
@@ -0,0 +1,51 @@
+import { useMediaQuery } from '@mui/material';
+import React from 'react';
+import { Navigation, Pagination } from 'swiper';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { SwiperModule } from 'swiper/types';
+import { theme } from '../../utils/theme';
+
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+
+type ModuleSelection = {
+  navigable?: boolean;
+  paginated?: boolean;
+};
+
+type Props = ModuleSelection & {
+  children: React.ReactNode[];
+};
+
+const moduleMap: {
+  [key: string]: SwiperModule;
+} = {
+  navigable: Navigation,
+  paginated: Pagination,
+};
+
+const convertSelectionToModules = (selection: ModuleSelection): SwiperModule[] => {
+  return Object.entries(selection)
+    .filter((entry) => entry[1])
+    .map(([prop]) => moduleMap[prop]);
+};
+
+export default function SwiperWrapper({ children, ...moduleSelection }: Props) {
+  const modules = convertSelectionToModules(moduleSelection);
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
+
+  return (
+    <Swiper
```

Comment:
<img width="361" alt="Screenshot 2022-10-16 at 1 11 43 PM" src="https://user-images.githubusercontent.com/71375383/196019215-e3c75807-c627-4cc7-8616-24b57602b139.png">

Swiper is buggy when cards are different height

Reply:
Fixed by Container